samwise.js

```diff
@@ -1,38 +1,39 @@
 /**
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.